code/modules/cargo/packs.dm

```diff
@@ -147,11 +147,27 @@
 	name = "NULL_ENTRY"
 	hidden = TRUE
 	cost = 14000
-	contains = list(/obj/item/weapon/storage/box/syndicate)
+	contains = null
```

Comment:
Just put in a dummy object to remove once you put the first syndicate gear in it.